src/_angular/font-setting/dtv-font-setting.js

```diff
@@ -158,8 +160,14 @@
                */
               skin_url: "//s3.amazonaws.com/rise-common/styles/tinymce/rise",
               statusbar: false,
-              toolbar: "fontselect fontsizeselect | alignleft aligncenter alignright alignjustify | forecolor backcolor | bold italic underline",
+              toolbar: "fontselect fontsizeselect | alignleft aligncenter alignright alignjustify" +
+              ($scope.verticalAlign ? " aligntop alignmiddle alignbottom" : "") +
```

Comment:
@donnapep Yes, only Spreadsheet requires it so far, so this setting is not always applicable. Thanks